variants/sidekiq/template.rb

```diff
@@ -37,7 +37,7 @@
       - redis
   redis:
     image: redis
-", after: /^services:$"/
+", after: /^services:\n/
```

Comment:
non-blocking: I usually prefer `\A` and `\z` as start and end delimiters in ruby regex because they don't change behaviour in multi-line strings